pom.xml

```diff
@@ -232,6 +232,7 @@
              Custom test jobs can override this using -D, e.g. -Dtestsuite.default.build.project.prefix=ee- -->
         <testsuite.default.build.project.prefix></testsuite.default.build.project.prefix>
         <wildfly.build.output.dir>${testsuite.default.build.project.prefix}build/target/${server.output.dir.prefix}-${server.output.dir.version}</wildfly.build.output.dir>
+        <wildfly.dist.output.dir>${testsuite.default.build.project.prefix}dist/target/${server.output.dir.prefix}-${server.output.dir.version}</wildfly.dist.output.dir>
```

Comment:
I don't see this used anywhere.

Reply:
Deleted, yes, it was unused.

I was left as cruft from development I forgot to remove; I thought I was going to need to use it but then I realized it was not necessary since `manualmode/pom.xml` and` manualmode-expansion/pom.xml` would run independently of this dynamic value (based on `testsuite.default.build.project.prefix`)